common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -235,6 +235,10 @@ $sequence--border-color: #C8C8C8;
   body.touch-based-device & ol li a:hover p {
     display: none;
   }
+
+  @media print {
```

Comment:
It's already hiding information about the chapter and sequence names that the page is printing from. I'd love to display all of it in the header, but I've no idea how I'd put it there without changing the page templates. Given that the present "sequential bar" method of navigation completely obscures all information that is present in each tab already, I don't know a good way to fix this. If the tabs were numbered on the screen, this would be a completely different story.